doc/source/examples/extended_examples/Krylov/krylov_example.rst

```diff
@@ -0,0 +1,378 @@
+
+Launch mapdl in ``'-smp'`` mode to avoid distributing processing which is not supported by the MAPDL Math module.
+
+.. code:: ipython3
+
+    mapdl = launch_mapdl(additional_switches='-smp', nproc=4)
+    mapdl.clear()
+    mm = mapdl.math
+    
+
+Parameters definition
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    pi = np.arccos(-1)
+    rho = 1.2                        # density
+    c0 = 340                         # Speed of Sound
+    frq = 1000                       # excitation freq   Hz
+    visco = 0.9                      # viscosity
+    
+    TP = 1/frq
+    WL = c0 * TP
+    no_wl = 3                        # no of wavelengths in space
+    
+    cyl_L = no_wl * WL               # length of duct
+    cyl_r = 0.025 * cyl_L            # cross section of duct
+    
+    nelem_wl = 10                    # no of elements per wavelength
+    tol_elem = nelem_wl * no_wl
+
+Element and Material definition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    mapdl.prep7()
+    
+    mapdl.et(1,'FLUID220', kop2=1)   # Uncoupled acoustic element without FSIs
+    mapdl.mp("DENS", 1, rho)
+    mapdl.mp("SONC", 1, c0)
+    mapdl.mp("VISC", 1, visco)
+
+.. parsed-literal::
+
+    MATERIAL          1     VISC =  0.9000000
+
+Geometry definition
+~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    mapdl.allsel()
+    mapdl.csys(0)
+    mapdl.wpcsys(-1)
+    mapdl.wprota(thzx=90)
+    mapdl.asel('NONE')
+    mapdl.vsel('NONE')
+    mapdl.cyl4(0, 0, cyl_r)
+    mapdl.wpcsys(-1)
+    mapdl.vext("ALL", dx=cyl_L)
+    mapdl.vsbw("ALL", keep='DELETE')
+    mapdl.wprota(thzx=90)
+    mapdl.vsbw("ALL", keep='DELETE')
+    mapdl.wpcsys(-1)
+    mapdl.cm('cm1', 'volu')
+
+
+.. parsed-literal::
+
+    DEFINITION OF COMPONENT = CM1       ENTITY=VOLU
+
+Mesh Creation
+
+.. code:: ipython3
+
+    mapdl.mat(1)
+    mapdl.type(1)
+    mapdl.cmsel("S", "cm1")
+    mapdl.aslv()
+    mapdl.lsla()
+    mapdl.lsel("U", 'loc', 'x', 0)
+    mapdl.lsel("U", 'loc', 'x', cyl_L)
+    mapdl.lesize('ALL',ndiv = tol_elem)
+    mapdl.lsla()
+    mapdl.vsweep('ALL')
+    mapdl.allsel()
+
+
+.. parsed-literal::
+
+    SELECT ALL ENTITIES OF TYPE= ALL  AND BELOW
+
+Plot FE model :
+
+.. code:: ipython3
+
+    mapdl.eplot()
+
+.. image:: ../../../examples/extended_examples/Krylov/Harmonic_Analysis_using_krylov_pymapdl_files/Harmonic_Analysis_using_krylov_pymapdl_15_1.png
+
+
+Boundary Condition Definition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+.. code:: ipython3
+
+    mapdl.cmsel("S", "cm1")
+    mapdl.aslv()
+    mapdl.asel('R',"EXT")
+    mapdl.asel('R',"LOC","x",0)
+    mapdl.nsla('S',1)   
+    mapdl.d('ALL','PRES', 1)
+    mapdl.cmsel("S", "cm1")
+    mapdl.aslv()
+    mapdl.asel('R',"EXT")
+    mapdl.asel('R',"LOC","x",cyl_L)
+    mapdl.nsla("S",1)      
+    mapdl.sf("ALL","IMPD",1000)
+    mapdl.allsel()
+
+.. parsed-literal::
+
+    SELECT ALL ENTITIES OF TYPE= ALL  AND BELOW
+
+
+Perform Modal Analysis to study the natural modes of the system
```

Comment:
```suggestion
Perform modal analysis to study the natural modes of the system
```

Reply:
This section title should probably be shorted to "Perform modal analysis" and the why placed in the section description.

Reply:
I agree.

Reply:
Title changed to "Perform modal analysis"